src/types.rs

```diff
@@ -46,16 +46,16 @@ impl EdgeKind {
 }
 
 /// Describes the edges required to/from a node. This includes both the concept of "signature" in the spec,
-/// and also the target (value) of a call (constant).
+/// and also the target (value) of a call (static).
 #[cfg_attr(feature = "pyo3", pyclass)]
 #[derive(Clone, Default, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub struct Signature {
     /// Value inputs of the function.
     pub input: TypeRow,
     /// Value outputs of the function.
     pub output: TypeRow,
-    /// Possible constE input (for call / load-constant).
-    pub const_input: TypeRow,
+    /// Possible static input (for call / load-constant).
+    pub static_input: TypeRow,
```

Comment:
Given we're renaming this, perhaps we should call it `static_inputs` as `TypeRow` allows more than one

Reply:
Oh, no, sorry, we call inputs `input`, fair enough